flow/templates/umich-greatlakes.sh

```diff
@@ -0,0 +1,31 @@
+{% extends "slurm.sh" %}
+{% block tasks %}
+{% set threshold = 0 if force else 0.9 %}
+{% set cpu_tasks = operations|calc_tasks('np', parallel, force) %}
+{% set gpu_tasks = operations|calc_tasks('ngpu', parallel, force) %}
+{% if gpu_tasks and 'gpu' not in partition and not force %}
+{% raise "Requesting GPUs requires a gpu partition!" %}
+{% endif %}
+{% set nn_cpu = cpu_tasks|calc_num_nodes(36) if 'gpu' not in partition else cpu_tasks|calc_num_nodes(40) %}
+{% set nn_gpu = gpu_tasks|calc_num_nodes(2) if 'gpu' in partition else 0 %}
+{% set nn = nn|default((nn_cpu, nn_gpu)|max, true) %}
+{% if partition == 'gpu' %}
+#SBATCH --nodes={{ nn|default(1, true)|check_utilization(gpu_tasks, 2, threshold, 'GPU') }}
+#SBATCH --ntasks-per-node={{ (gpu_tasks, cpu_tasks)|max }}
+#SBATCH --gres=gpu:{{ gpu_tasks }}
+{% else %}{# standard compute partition #}
+#SBATCH --partition=standard
+#SBATCH --nodes={{ nn|check_utilization(cpu_tasks, 36, threshold, 'CPU') }}
+#SBATCH --ntasks-per-node={{ (36, cpu_tasks)|min }}
+{% endif %}
+{% endblock tasks %}
+{% block header %}
+{{ super () -}}
+{% set account = account|default(environment|get_account_name, true) %}
+{% if account %}
+#SBATCH --account={{ account }}
+{% endif %}
+{% if memory %}
+#SBATCH --mem={{ memory }}
```

Comment:
Just want to make sure. Shall we include the unit of the memory here? Like `#SBATCH --mem={{ memory }}g` or `#SBATCH --mem={{ memory }}m`. Other parts look good to me. @Tobias-Dwyer

Reply:
Good question -- the unit (gigabytes) should be included by the user. The default is `4g`, and the submission help text shows that.